src/indexd/index/drivers/alchemy.py

```diff
@@ -119,8 +119,8 @@ def to_document_dict(self):
             metadata["release_number"] = self.release_number
 
         urls_metadata = extract_urls_metadata(self.urls_metadata)
-        created_date = self.created_date.isoformat()
-        updated_date = self.updated_date.isoformat()
+        created_date = self.created_date.isoformat() if self.created_date else None
+        updated_date = self.updated_date.isoformat() if self.updated_date else None
```

Comment:
The `created_date` and `updated_date` are actually nullable fields on the database